Katan/config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
+  get  'login' => 'sessions#new'
+  post 'login' => 'sessions#create'
+  delete 'logout' => 'sessions#destroy'
+
   get 'wssample/index'
   root 'tops#index'
+
+
```

Comment:
__[rubocop]__
* Extra blank line detected.
* Line is too long. [84/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->